Modules/Quotas/src/Quotas.Domain/Aggregates/Identities/Identity.cs

```diff
@@ -44,8 +44,8 @@ public Identity(string address, TierId tierId)
 
     public IndividualQuota CreateIndividualQuota(MetricKey metricKey, int max, QuotaPeriod period)
     {
-        if (max <= 0)
-            throw new DomainException(DomainErrors.MaxValueCannotBeLowerOrEqualToZero());
+        if (max < 0)
+            throw new DomainException(DomainErrors.MaxValueCannotBeLowerThanZero());
```

Comment:
Didn't we make this change in another PR already?

Reply:
I'm sure I only made this change here, it was last week though.

Reply:
Hmm, okay. Maybe I am remembering this from last week then.